server.js

```diff
@@ -10,4 +10,4 @@ app.get("/", (req, res) => {
   res.sendFile(__dirname + "/dist/index.html");
 });
 
-app.listen(port, console.log("Listening on port " + port));
+app.listen(port, console.log("STUMBL: Listening on port " + port));
```

Comment:
Nice touch!

Reply:
I think I added this at some point early on because vue doesn't auto-launch a chrome window and it was tough to find the listener in the terminal logs.  I honestly forget lol.